cmd/utils/nodecmd/defaultcmd.go

```diff
@@ -120,6 +110,17 @@ func startNode(ctx *cli.Context, stack *node.Node) {
 	} else {
 		startKlaytnAuxiliaryService(ctx, stack)
 	}
+
```

Comment:
You mean that the above code should be finished earlier than other code being executed in another goroutine. So, you move the following code at the bottom of the function, right? Can you explain details about the disorder of which goroutines makes the bad block issue? 
Your change seems that it can mitigate the issue somewhat, but cannot resolve it completely.

Reply:
I think this change is okay if we need lots of engineering to resolve the issue, but we need to figure out whether other race issues can occur because of the same reason.

Reply:
@aidan-kwon It seems that the node used uninitilialized istanbul chainConfig because sync starts before istanbul chainConfig is set.

The following functions are called to set istanbul chainConfig. startKlaytnAuxiliaryService -> cn.StartMining -> go s.miner.Start() -> self.worker.start() -> istanbul.Start -> sb.SetChain(chain)

At sb.SetChain(chain), `chainConfig` is set by setting `chain` to `sb.chain`. So, I changed the code to call `startKlaytnAuxiliaryService` earlier than the unlocking process.

Reply:
I cannot get the explanation. As far as I know, the moved code is not affected by chainConfig. What is the code using uninitilialized istanbul chainConfig?